espanol/noticias/burla-periodista/leones/leones.md

```diff
@@ -0,0 +1 @@
+El periodista queda rodeado por 4 expectaculares fieras que podrían destrozrlo tan solo al escuchar a Beatito chasquir sus dedos.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.